_data/projects/multy.yml

```diff
@@ -0,0 +1,14 @@
+name: Multy
+desc: Multy is a tool that enables developers to write cloud-agnostic configuration to deploy and switch to any cloud provider 
+site: https://github.com/multycloud/multy
+tags:
+  - go
+  - cloud
+  - aws
+  - azure
+  - terraform
+  - infrastructure-as-code
+  - multi-cloud
+upforgrabs:
+  name: good first issue
+  link: https://github.com/multycloud/multy/issues
```

Comment:
@danvdb18 This should be https://github.com/multycloud/multy/labels/good%20first%20issue